test/karma-unit.conf.js

```diff
@@ -1,80 +0,0 @@
-const pkg = require('../package.json');
-const mainLib = 'dist/' + pkg.name +'.js';
-
-module.exports = function(karma) {
-    karma.set({
-        // base path, that will be used to resolve files and exclude
-        basePath: '../',
-
-        preprocessors: {
-            'test/**/**/*.coffee': ['coffee'],
-        },
-        coffeePreprocessor: {
-            options: {
-                bare: true,
-                sourceMap: false
-            },
-            transformPath: function(path) {
-                return path.replace(/\.js$/, '.coffee');
-            }
-        },
-        // list of files / patterns to load in the browser
-        files: [
-            'bower_components/leaflet/dist/leaflet-src.js',
-            'bower_components/angular/angular.js',
-            'bower_components/angular-mocks/angular-mocks.js',
-            'bower_components/angular-simple-logger/dist/angular-simple-logger.js',//THIS IS BROWSER version
-            'bower_components/leaflet.markercluster/dist/leaflet.markercluster.js',
-            'bower_components/leaflet.vector-markers/dist/Leaflet.vector-markers.js',
-            mainLib,
-            'test/unit/bootstrap.coffee',
-            'test/unit/*.js',
-            'test/unit/**/*.js',
-            'test/unit/**/*.coffee',
-            'bower_components/Leaflet.PolylineDecorator/leaflet.polylineDecorator.js',
-            //do not include those specs for jasmine html runner by karma kama_jasmine_runner.html
-            {pattern:'**/**/**/*.coffee', included: false},
```

Comment:
This glob is what was crashing karma 0.13 and making so slow. It makes all files get read in like node_modules, and bower_comonents. Basically memory blows up. This is now fixed.